pom.xml

```diff
@@ -1910,43 +1941,42 @@
         <guava.version>18.0</guava.version>
         <hamcrest.version>1.3</hamcrest.version>
         <xmlunit.version>1.6</xmlunit.version>
-        <hk2.version>2.5.0-b42</hk2.version>
+        <hk2.version>2.5.0-b62</hk2.version>
+        <hk2.config.version>2.5.0-b56</hk2.config.version>
         <httpclient.version>4.5</httpclient.version> <!-- TODO: 4.5.2 doesn't work; apache client connector tests -->
-        <jackson.version>2.8.10</jackson.version>
+        <jackson.version>2.9.6</jackson.version>
         <jackson1.version>1.9.13</jackson1.version>
-        <javassist.version>3.22.0-CR2</javassist.version>
-        <javax.annotation.version>1.2</javax.annotation.version>
-        <javax.annotation.bundle.version>1.2</javax.annotation.bundle.version>
-        <javax.el.version>2.2.4</javax.el.version>
-        <javax.el.impl.version>2.2.4</javax.el.impl.version>
-        <javax.interceptor.version>1.2</javax.interceptor.version>
-        <javax.persistence.version>1.0</javax.persistence.version>
-        <javax.validation.api.version>1.1.0.Final</javax.validation.api.version>
+        <javassist.version>3.22.0-GA</javassist.version>
+        <javax.annotation.version>1.3.2</javax.annotation.version>
+        <javax.annotation.bundle.version>1.3.2</javax.annotation.bundle.version>
+        <javax.el.version>3.0.1-b06</javax.el.version>
+        <javax.el.impl.version>3.0.1-b09</javax.el.impl.version>
+        <javax.interceptor.version>1.2.2</javax.interceptor.version>
+        <javax.persistence.version>1.0.2</javax.persistence.version>
+        <javax.validation.api.version>2.0.1.Final</javax.validation.api.version>
         <jaxb.api.version>2.2.7</jaxb.api.version>
         <jaxb.ri.version>2.2.7</jaxb.ri.version>
         <jsonb.api.version>1.0</jsonb.api.version>
         <jaxrs.api.spec.version>2.1</jaxrs.api.spec.version>
-        <jaxrs.api.impl.version>2.1.1</jaxrs.api.impl.version>
```

Comment:
This is the version released lately. They should have a new version with jakarta group id soon, though.